sormas-api/src/main/java/de/symeda/sormas/api/FacadeProvider.java

```diff
@@ -407,7 +407,7 @@ public static ExternalLabResultsFacade getExternalLabResultsFacade() {
 				return (ExternalLabResultsFacade) get().ic.lookup(jndiName);
 			}
 		} catch (NamingException e) {
-			throw new RuntimeException(e.getMessage(), e);
```

Comment:
It doesn't make sense to catch the exception if you just trow it. Please remove the try catch.